internal/webhook/route_webhook.go

```diff
@@ -25,7 +25,10 @@ type RouteMutator struct {
 	Client  client.Client
 }
 
-const clusterIngressName = "cluster"
+const (
+	clusterIngressName = "cluster"
+	bypassLabel        = "haproxy.router.dana.io/bypass-mutation"
```

Comment:
For better clarity it'd be better to call this label `haproxy.router.dana.io/bypass-env-mutation` IMO